src/Shared/Recruit.Vacancies.Client/Application/CommandHandlers/ApproveVacancyReviewCommandHandler.cs

```diff
@@ -76,13 +83,21 @@ public async Task Handle(ApproveVacancyReviewCommand message, CancellationToken
             if (closureReason != null)
             {
                 await CloseVacancyAsync(vacancy, closureReason.Value);
+                await SendNotificationToEmployerAsync(vacancy.TrainingProvider.Ukprn.GetValueOrDefault(), vacancy.EmployerAccountId);
                 await _dashboardService.ReBuildDashboardAsync(vacancy.EmployerAccountId);
                 return;
             }
 
             await PublishVacancyReviewApprovedEventAsync(message, review);    
         }
 
+        private Task SendNotificationToEmployerAsync(long ukprn, string employerAccountId)
+        {
+            var communicationRequest = CommunicationRequestFactory.GetProviderBlockedEmployerNotificationForLiveVacanciesRequest(ukprn, employerAccountId);
```

Comment:
`TryGetReasonToCloseVacancy` suggests the ClosureReason could be TransferredByEmployer, TransferredByQa or BlockedByQa.  Is this communication appropriate for each of these reasons?